### ISSUE TITLE
Plan to create new release

### ISSUE THREAD

**Issue by ssauriera** [katoid/angular-grid-layout#57]
There is any plan to create a new release for angular 13 or 14?

**Comment by llorenspujol** [comment]
The current release is compatible with angular v12, v13 and v14, so I have not plans to make any release. Also there have been no new features. If there is something I don't know that making the release with the current angular v14 would improve something, I would consider for sure. But right now, the reality is that if I make a version with this current version, angular v14, users of the library using angular12 and angular13 would have to download the previous version (the current one's), since the library build with angular14 is not compatible with lower versions...

In conclusion, making a new version with  angular14, with no new features, seems a regression, because angular12 and angular13 users would need to specifically download the previous version, that is this one.

Would be nice to make a feature, and push a new release with angular v14 BTW.

**Comment by llorenspujol** [comment]
@ssauriera De res! (Si fas servir la lib, i necesites alguna 'feature', estem oberts a que la gent contribueixi, només per que ho sàpigues per si un cas)
